### PR TITLE
Added skip sync queue feature

### PR DESCRIPTION
**Pull request by linfeng2011**
Added feature to skip sync queue on postInsertObjectData function

### CODE REVIEW

File: interceptors/DataApiInterceptors.cfc [hunk 1]
```diff
@@ -125,13 +125,12 @@ component extends="coldbox.system.Interceptor" {
...
 
-	public void function postUpdateObjectData( event, interceptData ) {
-		if ( !_applicationLoaded ) return;
-		dataApiQueueService.queueUpdate( argumentCollection=interceptData );
-	}
-
 	public void function postInsertObjectData( event, interceptData ) {
 		if ( !_applicationLoaded ) return;
+
+		var skipQueue = IsBoolean( interceptData.skipSyncQueue ?: "" ) && interceptData.skipSyncQueue;
```
Comment (DominicWatson) [inline interceptors/DataApiInterceptors.cfc]: I think @linfeng2011 that this should be `skipDataApiQueue` to be more precise. Can you change it?

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Nice one, thanks @linfeng2011. Could you tweak the naming as per my comment? And also add this to the queue documentation in the README?

File: interceptors/DataApiInterceptors.cfc (hunk 1 above)
Comment (linfeng2011) [reply interceptors/DataApiInterceptors.cfc]: Hey @DominicWatson , thanks for reviewing. I have change the variable name and added documentation to README accordingly, thanks!

**Comment by DominicWatson** [comment]
Nice one, thanks @linfeng2011 :)

**Comment by DominicWatson** [comment]
Merged and released in v3.0.10

**Pull request merged**